packages/config/config/devices/0x000c/hs-fls100-g2.json

```diff
@@ -40,29 +40,30 @@
 			"allowManualEntry": false,
 			"options": [
 				{
-					"label": "Load controlled by Z-Wave only",
+					"label": "Z-Wave only",
 					"value": 0
 				},
 				{
-					"label": "Load controlled by Z-Wave and Sensors",
+					"label": "Z-Wave and Sensors",
```

Comment:
We generally try to avoid repetition between the parameter label and option labels